tests/ui/fmt/help-message-issue-129966-reorder-remove-format-specifier.rs

```diff
@@ -0,0 +1,14 @@
+// Test checks for the help messages in the error output/
+//@ check-run-results
+
+struct Foo(u8, u8);
+
+fn main() {
+    let f = Foo(1, 2);
+    format!("{f:?#}");
+//~^ ERROR invalid format string: unknown format identifier '?#'
```

Comment:
Nit: these UI test error annotations do not care about preceding indent whitespace, they match on a line-by-line basis. Usually we ident them in alignment with surrounding context as suitable.

```suggestion
    //~^ ERROR invalid format string: unknown format identifier '?#'
```